### ISSUE TITLE
[TF2] TF2 Suffering Audio Issues

### ISSUE THREAD

**Issue by pavUed000** [ValveSoftware/Source-1-Games#5742]
Entering a match now has a chance to be fully audio-corrupted, making it sound like a chainsaw

**Comment by Joshua-Ashton** [comment]
Have you changed your snd_mixahead value?

**Comment by pavUed000** [comment]
> Have you changed your snd_mixahead value?

Yep, still had some issues tho. Ended up re-installing tf2 and the problem seems to be fixed.

**Comment by Pikachara** [comment]
> > Have you changed your snd_mixahead value?
> 
> Yep, still had some issues tho. Ended up re-installing tf2 and the problem seems to be fixed.

In my case reinstalling didn't help.
Putting the files thru converter also didn't help.
i deleted soundcache files which apperently should help will write later if it helps. if it didn't i just won't write anything :p
